### DISCUSSION TITLE
How to entirely turn off prettier and setup my own formatting rules

### DISCUSSION THREAD

**Discussion by theArianit** [zed-industries/zed#15672] [Feature Requests]
As the title says. But about the custom formatting, there seem to be lacking documentation. There is not an easy way to find the settings that you can configure. I want to turn of the automatic ; in css properties but can't find any list of configs that can be applied.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
